cmd/utils/utils.go

```diff
@@ -0,0 +1,13 @@
+package utils
```

Comment:
utils is a bad package name, it doesn't say what it includes, also this file includes two items that have nothing in common. I suggest maybe to put in root.go

Reply:
This should go away in a follow up PR as there will be no use of globals, right now it's not possible to put it in root because it will cause cyclic import.

Regarding the subpackages - the idea is to have a package per sub command so all policy related commands under `policy` and so on. especially as looks like we are growing `cloudquery` CLI interface.